test/cypress/integration/group-paying.spec.js

```diff
@@ -1,7 +1,16 @@
+
+// Similar to time.js but without the import.
+export function humanDate (datems, opts = { month: 'short', day: 'numeric' }) {
+  const locale = navigator.languages ? navigator.languages[0] : navigator.language
+  return new Date(datems).toLocaleDateString(locale, { ...opts, timeZone: 'UTC' })
```

Comment:
If you're doing this here in the tests, then the change that was done to `humanDate` in the other file in the other PR needs to be reverted.

Reply:
What change? It needs to be as it is

Reply:
This change: https://github.com/okTurtles/group-income-simple/pull/918/files#diff-eb95d40daae3bff68bee21866026301eR63

```js
  const timezone = window.cypress ? { timeZone: 'UTC' } : {} // ensure tests work consistently anywhere in the world.
  return new Date(datems).toLocaleDateString(locale, { ...opts, ...timezone })
```

Keep the `humanDate` as it was before, and in the tests, display it without `timeZone: 'UTC'`.

So above, remove `timeZone: 'UTC'`, and revert the prior change. This will result in the time being displayed in the browser being in local timezone to whatever machine is running the tests, and it will make sure the string that's being compared in the tests is also local timezone.

Reply:
The timezone will be the same in both scenarios because it’s run through cypress.

Reply:
Yes, so there is no need for `humanDate` to perform this extra computation by checking for `window.cypress` and destructuring/restructuring an object.

Reply:
It has because humanTime at time.js can be run outside cypress. If we remove it, the test will fail in your timezone. (Not now, but in a few hours when it’s a different day between here and in pacific time)

Reply:
I did this locally. Removing the `timeZone` parameter keeps both strings displaying in my local timezone. So there is no problem. You don't need to display the time in UTC, you just need to make sure that the timezones match. The problem before was that they weren't matching because you had hand-written `'Feb 10'`. Since it's no longer being hand-written, there should be no problem.

Reply:
ahh, I finally understood what you meant. Changed!